docs/source/user/installing.rst

```diff
@@ -262,7 +262,9 @@ information.
 +----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
 | Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
 +===========+==============+===========================+============================+==============================+===================+=============================+
-| 0.58.0    | 2023-09-20   | 3.8.x <= version < 3.12   | 1.21 <= version < 1.26     | 0.41.x                       | 14.x              | 2021.6 <= version           |
+| 0.58.1    | UNRELEASED   | 3.8.x <= version < 3.12   | 1.22 <= version <= 1.26    | 0.41.x                       | 14.x              | 2021.6 <= version           |
```

Comment:
```suggestion
| 0.58.1    | UNRELEASED   | 3.8.x <= version < 3.12   | 1.22 <= version < 1.27    | 0.41.x                       | 14.x              | 2021.6 <= version           |
```

Reply:
Such that we use the same format in the version support table. I don't think entries should vary in the way in which the specify upper and lower bounds. Lower bound inclusive and upper bound exclusive seems like what has been done until now.